log4j-core/src/main/java/org/apache/logging/log4j/core/config/LoggerConfig.java

```diff
@@ -649,7 +649,7 @@ public boolean requiresLocation() {
         return false;
     }
 
-    private void logParent(final LogEvent event, final LoggerConfigPredicate predicate) {
+    private void logParent(final LogEvent event, final DispatchType predicate) {
```

Comment:
Could you rename `predicate` to `dispatchType`, please?

Reply:
In the corrected version I use `Predicate<LoggerConfig>`.